cmd/keeper/keeper.go

```diff
@@ -103,7 +107,21 @@ var defaultPGParameters = pg.Parameters{
 	"hot_standby":             "on",
 }
 
-func (p *PostgresKeeper) getSUConnParams(keeperState *cluster.KeeperState, setPassword bool) pg.ConnParams {
+func (p *PostgresKeeper) getInitialSUPassword() string {
+	var pw string
+
+	if p.cfg.initialSUPasswordFile != "" {
+		pwBytes, err := ioutil.ReadFile(p.cfg.initialSUPasswordFile)
```

Comment:
should we check file permissions to be <= than 600. Perhaps that's a lesser problem inside a pod but on a standard server it'll be good to make them readable only by the stolon's owner.

But looks like that kubernetes actually doesn't implement setting file permission so actually it should ignore the check on kubernetes. Perhaps just add a TODO at the moment...
Haven't tried so I'm not sure it will be possible to them with that user / permissions using kubernetes secrets?